Formula/s/simutrans.rb

```diff
@@ -1,10 +1,9 @@
 class Simutrans < Formula
   desc "Transport simulator"
   homepage "https://www.simutrans.com/"
-  url "svn://servers.simutrans.org/simutrans/trunk/", revision: "11395"
-  version "124.2.2"
+  url "svn://servers.simutrans.org/simutrans/trunk/", revision: "11588"
```

Comment:
Revision for Simutrans 124.3 is finally 11590, see https://forum.simutrans.com/index.php?msg=209830